helm/adelphi/templates/cassandra-clusters.yaml

```diff
@@ -77,7 +77,9 @@ spec:
           clusterName: {{ .Values.target.clusterName }}
           serverType: cassandra
           serverVersion: {{ .Values.target.version }}
+          {{- if .Values.target.commit_hash }}
           serverImage: {{`{{inputs.parameters.registry_ip}}`}}:30000/adelphi/management-api-for-apache-cassandra
```

Comment:
Is this... right?  Shouldn't this be the C* image and not the management API image?

Reply:
Yes, it's correct, this has to be the MGMT API image. If you try it on k3d or GKE you'll see the C* cluster will be provisioned with the correct git hash version.